config/cluster-install/rbac/numaflow-server-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
I have added the clusterRoleBinding and the clusterRole, but re-used the existing ones. Let me know if it is good practice to add new cluster roles for server

Reply:
Please use a separate set of sa, role and binding.